src/lib/__tests__/validators.ts

```diff
@@ -59,3 +59,37 @@ describe('validateEmail', () => {
     });
   });
 });
+
+describe('Validate The valid displayName to be true', () => {
+  [
+    'Ähtö',
+    'Aki',
+    'Äteritsiputeritsipuolilautatsijänkä',
+    'Veera',
+    'Ivan',
+    'eppu',
+    'feelix',
+    'pyry',
+  ].forEach(validName => {
+    it(`decodes valid displayName ${validName}`, () => {
+      expect(isRight(validators.ValidName.decode(validName))).toEqual(true);
+    });
+  });
+});
+
+describe('Validate The invalid displayName to be true', () => {
+  [
+    'Ähtö@',
+    'Aki©',
+    'Äteritsiputeritsipuolilautatsijänkädsadasd',
+    'Veera..',
+    'Ivan)',
+    'eppu=',
+    'feelix!"',
+    'pyry!"#',
```

Comment:
Maybe add emtpy and too short string here. Probably string with an emoji too?

Reply:
Ok bro i will thanx for your review